pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -153,6 +153,14 @@ buffer=/var/spool/wavefront-proxy/buffer
 configuration in Istio.
 #traceZipkinListenerPorts=9411
 
+## The following settings are used to configure trace data sampling:
+## The rate for traces to be sampled. Can be from 0.0 to 1.0. Defaults to 1.0
+## Ex: The below property sets up a sampling rate of 10%. A sampling rate of 10% causes 1 out of 10 traces to be sent and ingested.
+#traceSamplingRate=.1
```

Comment:
Let's set them to the default values even when commented out: `1.0` for rate and `0` for duration. (can remove the example part).